.github/workflows/CI-unix.yml

```diff
@@ -126,3 +126,36 @@ jobs:
       - name: Test
         run: |
           ${{ matrix.config.qemu }} build/uv_run_tests_a
+
+  build-bsd:
+    timeout-minutes: 30
+    runs-on: ${{ matrix.config.runner }}
+    name: build-${{ matrix.config.os }}-${{ matrix.config.version }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          # The OS versions supported are specific to the version of the action
+          # https://github.com/cross-platform-actions/action/blob/master/changelog.md
+          - { os: freebsd, version: '13.1', runner: macos-11, install: 'pkg install -y' }
+          - { os: openbsd, version: '7.2',  runner: macos-11, install: 'pkg_add -I' }
```

Comment:
Can we use ubuntu-latest here? There is generally less availability of macos runners. The cross-platform-actions seems to imply it is operational

Reply:
The ubuntu runners use qemu, which I figured would be slower than the macOS hypervisor.  I'll give it a shot and see.

Reply:
FreeBSD seems fine with the ubuntu runner, but OpenBSD was hitting the 30m timeout when it passes in ~10 minutes on macos-11 runner.